source/collection-management/tests/modifyCollection-errorResponse.yml

```diff
@@ -5,6 +5,7 @@ schemaVersion: "1.12"
 createEntities:
   - client:
       id: &client0 client0
+      useMultipleMongoses: false
```

Comment:
Was this necessary? This test never used `configureFailPoint`.

I notice you didn't make any changes to `aggregate-merge-errorResponse.yml`; however, that test was already skipping sharded clusters.

Reply:
Ah it's not, great point. We don't actually run this test in the Go driver (see [GODRIVER-2587](https://jira.mongodb.org/browse/GODRIVER-2587)), so I just added it without thinking. I did _not_ add `useMultipleMongoses: false` to aggregate-merge-errorResponse` because it skips sharded clusters.

Reply:
@benjirewis: I think `useMultipleMongoses: false` was erroneously added to the `findOneAndUpdate` test, too, which doesn't use `configureFailPoint`. See: https://github.com/mongodb/specifications/pull/1329